unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotsView.cs

```diff
@@ -15,7 +15,7 @@ public class AvatarSlotsView : BaseComponentView, IAvatarSlotsView
         [SerializeField] private GameObject sectionSeparator;
 
         private readonly Dictionary<string, Transform> avatarSlotSections = new ();
-        private readonly Dictionary<string, IAvatarSlotComponentView> avatarSlots = new ();
+        internal readonly Dictionary<string, IAvatarSlotComponentView> avatarSlots = new ();
```

Comment:
I'm seeing this variable is not being using out of this class, right? Why do we need it to be internal?

Reply:
+1

Reply:
Done :)